### PR TITLE
Linwrap 903 prime

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

**Comment by UnixJunkie** [comment]
resurrection of https://github.com/ocaml/opam-repository/pull/18445

File: packages/linwrap/linwrap.9.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,85 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/linwrap"
+bug-reports: "https://github.com/UnixJunkie/linwrap/issues"
+dev-repo: "git+https://github.com/UnixJunkie/linwrap.git"
+license: "BSD-3"
```
Comment (mseri) [inline packages/linwrap/linwrap.9.0.3/opam:7]:
```suggestion
license: "BSD-3-Clause"
```

Comment (UnixJunkie) [reply packages/linwrap/linwrap.9.0.3/opam:7]: ok, sure!

**Comment by mseri** [comment]
Thanks!

**Pull request merged**